styles/cinny/catppuccin.user.css

```diff
@@ -226,15 +226,15 @@
     body.prism-light,
     &.prism-light,
     &.prism-dark {
-      --prism-comment: @overlay0;
-      --prism-punctuation: @sky;
+      --prism-comment: @overlay2;
+      --prism-punctuation: @overlay2;
       --prism-property: @yellow;
       --prism-boolean: @peach;
       --prism-selector: @green;
-      --prism-operator: @sky;
```

Comment:
I thought we were keeping it sky?

Reply:
Right, this was the first one I fixed before I noticed 😅